pkg/logflags/logflags.go

```diff
@@ -177,8 +180,11 @@ func Setup(logFlag bool, logstr string, logDest string) error {
 			fnCall = true
 		case "minidump":
 			minidump = true
-			// If adding another value, do make sure to
-			// update "Help about logging flags" in commands.go.
+		default:
+			if !unKnownHasPrint { // Just print first unknown component.
+				fmt.Fprintf(os.Stderr, "Warning: unknown %q for '--log-output', run 'dlv help log' for usage.\n", logcmd)
```

Comment:
nit: `Warning: unknown log output value %s for ...`

Reply:
I remove `for '--log-output'` that the same meaning as `log output value` , avoid too long one line.